src/main/webapp/app/exam/participate/summary/result-overview/exam-result-overview.component.html

```diff
@@ -2,125 +2,127 @@
+                            <div *ngIf="exercise.maxPoints != undefined">
+                                {{ exercise.includedInOverallScore === IncludedInOverallScore.INCLUDED_AS_BONUS ? 0 : exercise.maxPoints }}
+                            </div>
+                        </td>
+                        <td class="text-center">
+                            <div
+                                *ngIf="exerciseInfos[exercise.id!].achievedPercentage !== undefined; else noPercentageKnownPlaceholder"
+                                [class]="exerciseInfos[exercise.id!].colorClass"
+                            >
+                                {{ exerciseInfos[exercise.id!].achievedPercentage }}%
+                            </div>
+                            <ng-template #noPercentageKnownPlaceholder> - </ng-template>
+                        </td>
+                        <td *ngIf="studentExamWithGrade?.maxBonusPoints">
+                            <div *ngIf="exercise.bonusPoints != undefined">
+                                {{ exercise.includedInOverallScore === IncludedInOverallScore.INCLUDED_AS_BONUS ? exercise.maxPoints : exercise.bonusPoints }}
+                            </div>
+                        </td>
+                    </tr>
+                </tbody>
+                <tfoot class="total-footer">
+                    <tr>
+                        <th scope="row" colspan="2">{{ 'artemisApp.exam.examSummary.points.total' | artemisTranslate }}</th>
+                        <td *ngIf="showIncludedInScoreColumn" class="text-center">-</td>
+                        <th class="text-center">
+                            {{ overallAchievedPoints }}
+                        </th>
+                        <th class="text-center">
+                            {{ maxPoints }}
+                        </th>
+                        <th class="text-center">
+                            <div
+                                *ngIf="studentExamWithGrade.studentResult.overallScoreAchieved !== undefined; else totalScoreNotDefinedPlaceholder"
+                                [class]="hasPassed ? 'text-success' : 'text-danger'"
+                            >
+                                {{ overallAchievedPercentageRoundedByCourseSettings }}%
+                            </div>
+                            <ng-template #totalScoreNotDefinedPlaceholder>
+                                <div>-</div>
+                            </ng-template>
+                        </th>
+                        <th *ngIf="studentExamWithGrade?.maxBonusPoints">
+                            {{ studentExamWithGrade?.maxBonusPoints }}
+                        </th>
+                    </tr>
+                </tfoot>
+            </table>
+            <div class="d-flex align-items-center">
+                <div class="you-achieved-container">
+                    <div *ngIf="studentExamWithGrade?.maxBonusPoints" class="h5">
```

Comment:
Changed the `h5` to only styling, to improve the metadata structure at least a bit